src/wesnoth_lua_config.md

```diff
@@ -0,0 +1,109 @@
+
+# Lua for Wesnoth #
+This document describes the process used to install Lua 5.3.3 for Wesnoth.
+
+The goal of this process was, as much as possible, install clean, unchanged sources.
+Traditionally, Wesnoth maintainers have made changes directly to the Lua source kit.
+__This is strongly discouraged.__
+Future maintainers should strive, as much as possible, to follow a simliar process.
+
+Future maintainers are expected to update this document as appropriate.
+
+## Before you begin ##
+
+Be sure you are using a copy of the current master.
+And be sure you are working in a private branch.
+
+        $ cd ~/wesnoth
+        $ git checkout master
+        $ git pull --rebase upstream master
+        $ git checkout -b Upgrade_to_Lua-5.3.3
+
+## Install Lua ##
+Download the current source kit from [the Lua maintainers](http://www.lua.org).
+For Lua 5.3.3, this was <http://www.lua.org/ftp/lua-5.3.3.tag.gz>.
+The following presumes you are working on Unix.
+Windows is a bit more work, but generally follows the same process.
+
+        $ cd ~
+        $ wget http://www.lua.org/ftp/lua-5.3.3.tar.gz
```

Comment:
This doesn't really matter, but also showing a `curl` command might help an occasional OSX user (which, last I checked, doesn't ship with `wget`).

That said, most OSX users would download it through the browser instead.